src/effects/effectchainmanager.h

```diff
@@ -53,6 +53,8 @@ class EffectChainManager : public QObject {
     bool saveEffectChains();
     QList<std::pair<EffectChainPointer, QDomElement>> loadEffectChains();
 
+    static const int kNumEffectsPerUnit = 4;
```

Comment:
I am not sure if this works nice on all compilers. 
I would prefer to wrap this into a getter and move this constant to the cpp file